### DISCUSSION TITLE
Does pymotif work with py2neo

### DISCUSSION THREAD

**Discussion by timlrx** [cylynx/motif.gl#166] [Q&A] [answered]
The demo shows pymotif with the neo4j library. Does it also work with [py2neo](https://py2neo.org/2021.1/)?

Py2neo is a client library and toolkit for working with Neo4j from within Python applications and from the command line. The library supports both Bolt and HTTP and provides a high level API, an OGM, admin tools, an interactive console, a Cypher lexer for Pygments, and many other bells and whistles.

**Comment by jasonho-lynx** [comment, marked as answer]
Not at the moment, but if you have a use-case that would benefit from having this integration, do let us know and we will look into it!
